content/cookbook/wallets/restore-keypair.md

```diff
@@ -22,9 +45,34 @@ const keypair = Keypair.fromSecretKey(
 );
 ```
 
+</Tab>
+
+</Tabs>
+
 ## From base58 String
 
-```typescript filename="restore-keypair-from-base58.ts
+<Tabs groupId="language" items={['web3.js v2 from base58', 'web3.js v1 from base58']}>
```

Comment:
please use the same groupId and item labels. This will make it so the user will only have to switch it once on the page, and all the tabs will update. better ux

```suggestion
<Tabs groupId="language" items={['web3.js v2', 'web3.js v1']}>
```

Reply:
i find that personally to be janky but all good